test-plans/ursa-network/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "ursa-network"
```

Comment:
Is the folder `/ursa-network/` necessary? Also maybe we call this package `ursa-network-tests`?

Reply:
This branch/draft is just for experimenting with `testground`, the plan is for us (the devs on working on this) to write a set of tests. Since we need different binaries for each test plan, it makes sense to seperate them per crate but we should definitely attempt to consolidate before merging.

Reply:
Oh I see, I just saw the crates folder so that makes sense to have a folder here per binary